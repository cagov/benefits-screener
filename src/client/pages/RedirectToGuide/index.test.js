import React from "react";
import Component from "./index";

describe("<RedirectToGuide />", () => {
  it("renders a redirect to the guide", async () => {
    expect(<Component />).toMatchSnapshot();
  });
});
