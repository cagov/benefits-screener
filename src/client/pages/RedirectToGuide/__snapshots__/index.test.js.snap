// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RedirectToGuide /> renders a redirect to the guide 1`] = `<RedirectToGuide />`;
